client/src/Components/Signup.jsx

```diff
@@ -1,7 +1,13 @@
-import { Button, TextField } from "@mui/material";
+import { Button, TextField, Typography } from "@mui/material";
 import React from "react";
+import GoogleIcon from "@mui/icons-material/Google";
+
 
 const Signup = () => {
+	const googleLogin = () => {
+		window.open("http://localhost:4000/auth/google", "_self");
+	};
+
 	return (
 		<div>
 			<form action='#'>
```

Comment:
I'm confused by this portion. I thought we were using Google o-auth to login? If so why do we need signup form?

Reply:
we don't need it to sign up but we can add it if we want to.